sklearn/svm/classes.py

```diff
@@ -906,6 +906,19 @@ class SVR(BaseLibSVM, RegressorMixin):
         `coef_` is readonly property derived from `dual_coef_` and
         `support_vectors_`.
 
+    class_weight_ : array, shape = [n_class]
+        Multipliers of parameter C of class i (class_weight_[i]*C).
+        Computed based on the ``class_weight`` parameter.
+
+    fit_status_ : int
+        0 if correctly fitted, 1 otherwise (will raise warning)
+
+    n_support_ : array-like, dtype=int32, shape = [n_class]
```

Comment:
there is no n_class for regression, can you check the shape?

Reply:
n_support_ is an array where in the example i used to check this, has shape: (2,) 
an example with -- n_samples, n_features = 10, 5 pulled from https://scikit-learn.org/stable/modules/generated/sklearn.svm.SVR.html

Reply:
looks like the content is garbage though :(

Reply:
Same bug as #14774 I supposed.

Reply:
yes -- i removed it from this PR

Reply:
https://github.com/scikit-learn/scikit-learn/issues/14788